pkg/kp/rc.go

```diff
@@ -0,0 +1,14 @@
+package kp
+
+import (
+	"github.com/square/p2/pkg/labels"
+	"github.com/square/p2/pkg/pods"
+)
+
+type ReplicationControllerStore interface {
+	Create(manifest pods.Manifest, nodeSelector, podSelector labels.Selector) (string, error)
```

Comment:
oh I think I did mean labels, not label selector. OK that will differentiate them well enough.